Apply includeSummaryInBody flag from SLA overrides to all schools [SCHOOL-108]

BillClinton opened the pull request:
In addressing `SCHOOL-108 Section pages have blank titles` it was discovered that the blank title issue had already been addressed but SLA had previous overrides so they were not seeing the fix.  These overrides introduced the `includeSummaryInBody` flag for blog posts. It was decided that the SLA behavior should be preserved and applied to all schools.

Once this PR is merged, the following files can be removed from SlateFoundation/scienceleadership-skeleton:
 - html-templates/blog/blogPost.tpl
 - html-templates/subtemplates/blog.tpl 
 - html-templates/sections/courseSection.tpl

BillClinton committed:
fix: add `includeSummaryInBody` flag from SLA

In addressing `SCHOOL-108 Section pages have blank titles` it was discovered that the
blank title issue had already been addressed but SLA had previous overides so were not
seeing the fix.  These overrides introduced the `includeSummaryInBody` flag for blog posts.
It was decided that the SLA behavior should be preserved and applied to all schools.

themightychris commented on html-templates/subtemplates/blog.tpl:
```diff
@@ -0,0 +1,78 @@
+{load_templates "subtemplates/people.tpl"}
```
@BillClinton PR this revised file all the way up to emergence-skeleton, it's just adding a new option that is off by default so that's no reason to override the whole thing in this project

BillClinton replied:
done. See this PR: https://github.com/JarvusInnovations/emergence-skeleton/pull/326

BillClinton committed:
fix: remove blog template to provided by Emergence

The minor changes this file was implementing have been moved to emergence-skeleton
in this PR: https://github.com/JarvusInnovations/emergence-skeleton/pull/326

themightychris commented on html-templates/blog/blogPost.tpl:
```diff
@@ -0,0 +1,8 @@
+{extends designs/site.tpl}
+
+{block "title"}{$data->Title} &mdash; {$dwoo.parent}{/block}
```
`$data->Title` needs an `|escape`

themightychris committed:
fix: add missing escape for blog post title

themightychris reviewed (approved):
one tweak which was a problem from upstream, I went ahead and made
